documentation/CHANGELOG_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+# Changelog API Sub Project name
+
+<!--NOTE: in above title replace "API Sub Project name" with actual API Sub Project name and remove this line-->
+
+## Table of content
+
+- **[r1.1 - alpha release](#r11---alpha-release)**
+
+**Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
+
+The below sections record the changes for each API version in each release as follows:
+
+* for an alpha release, the delta with respect to the previous release
+* for the first release candidate, all changes since the last public release
+* for subsequent release candidate(s), only the delta to the previous release candidate
+* for a public release, the consolidated changes since the previous public release
+
+<!-- Repeat the below release section (header 1 and subsections) at the top of this file for each new release-->
+
+# r1.1 - alpha release
+
+## Release Notes
+
+* API-name1 vx.y.z
+* API-name2 vx.y.z
+* Commonalities vx.y.z
+* ICM vx.y.z
+* Backward compatible (wrt vx.y.z rm.n): <!-- one of: yes | **no** | N/A (for alpha and release-candidates)-->
```

Comment:
See my comments and examples within the CHANGELOG example.

Especially the line Backward compatible is wrong here. Not a release is backward compatible, but an API version compared to another. It could be that API1 is backward compatible, but API2 not.

Reply:
removed and adapted - commonalities and ICM only kept in the initial public release and the patch release (as there could be a patch to one of those as well)